src/api/tasks/TaskRouter.router.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const { Router } = require("express");
+const { fail } = require("../../utils/express/response");
+
+const {
+  sendRegistrationToTascomiAction,
+  sendNotificationsForRegistrationAction,
+  saveRegistrationToTempStoreAction,
+  sendAllOutstandingRegistrationsToTascomiAction,
+  sendAllNotificationsForRegistrationsAction,
+  saveAllOutstandingRegistrationsToTempStoreAction
+} = require("./Tasks.controller");
+
+// const { viewDeleteRegistrationAuth } = require("../../middleware/authHandler");
+
+const TaskRouter = () => {
+  const router = Router();
+
+  //apply any middleware
```

Comment:
remove?

Reply:
dunno mate - you need these endpoints securing pal? I figured probably...

Reply:
depends if the end point is safe from the public :) - let me know mate

Reply:
it is safe since whitelisted but actually, there's no harm it putting it in.

Reply:
cool on it

Reply:
ST ANW decided to leave as comment for now - will unrem at later point.